Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    primary_connect_proto (0.8.1)
+    primary_connect_proto (0.14.0)
```

Comment:
Remind me why this doesn't match the `VERSION`?

Reply:
My bad, I didn't bundle after the changes. I haven't push to rubygems either yet, planning to do that after merging.